modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts

```diff
@@ -563,123 +602,83 @@ fragmentTest.describe('Visualization Modes in the fragment', () => {
-			visualizationModesPage.page.getByLabel('Sortable');
-
-		await expect(sortableInput).toBeInViewport();
-		await expect(sortableInput).toBeEnabled();
-		await expect(sortableInput).toBeChecked();
-
-		await sortableInput.click();
-
-		await expect(sortableInput).not.toBeChecked();
-
-		await visualizationModesPage.saveAddFieldsModal();
-
-		await expect(
-			visualizationModesPage
-				.getRowByText(SAMPLE_SCALAR_FIELD)
-				.locator('td')
-				.nth(SORTABLE_COLUMN_INDEX)
-		).toHaveText('false');
-	});
+	fragmentTest(
+		'Show mapped scalar array field in the fragment @LPD-11769',
+		async ({
+			apiHelpers,
+			dataSetManagerApiHelpers,
+			fdsFragmentPage,
+			page,
+			site,
+		}) => {
+			const SAMPLE_SCALAR_ARRAY_FIELD = 'keywords';
+			const SAMPLE_SCALAR_ARRAY_CONTENT = ['one', 'two', 'three'];
 
-	await test.step('Check if object field has disabled sortable option', async () => {
-		await visualizationModesPage
-			.getRowByText(`${SAMPLE_OBJECT_FIELD}.*`)
-			.locator('.actions-cell button')
-			.click();
+			await fragmentTest.step('Create table fields', async () => {
+				await dataSetManagerApiHelpers.createDataSetViewFields({
+					extraBodyParams: {
+						keywords: SAMPLE_SCALAR_ARRAY_CONTENT,
+					},
+					label_i18n: {en_US: SAMPLE_SCALAR_ARRAY_FIELD},
+					name: SAMPLE_SCALAR_ARRAY_FIELD,
+					r_fdsViewFDSFieldRelationship_c_fdsViewERC: viewERC,
+					type: 'array',
+				});
+			});
 
-		const editButton = visualizationModesPage.page.getByRole('menuitem', {
-			name: 'Edit',
-		});
+			const layout = await fragmentTest.step(
+				'Create a new page',
+				async () => {
+					const pageLayout =
+						await apiHelpers.headlessDelivery.createSitePage({
+							siteId: site.id,
+							title: getRandomString(),
+						});
 
-		await expect(editButton).toBeInViewport();
+					return pageLayout;
+				}
+			);
 
-		await editButton.click();
+			await fragmentTest.step(
+				'Configure Data Set in the page',
+				async () => {
+					await fdsFragmentPage.configureDataSetFragment({
+						layout,
+						site,
+						viewLabel,
+					});
+				}
+			);
 
-		const sortableLabel =
-			visualizationModesPage.page.getByLabel('Sortable');
+			await fragmentTest.step(
+				'Data Set Table is in the page',
+				async () => {
+					await fdsFragmentPage.fdsTableWrapper.waitFor({
+						state: 'visible',
+					});
 
-		await expect(sortableLabel).toBeInViewport();
+					expect(
+						await fdsFragmentPage.fdsTableWrapper
+					).toBeInViewport();
 
-		await expect(sortableLabel).toBeDisabled();
+					expect(
+						await page
+							.locator('.dnd-thead > div')
+							.first()
+							.locator('.dnd-th')
+							.allInnerTexts()
+					).toEqual([SAMPLE_SCALAR_ARRAY_FIELD, '']);
 
-		await visualizationModesPage.cancelAddFieldsModal();
-	});
+					expect(
+						await page
+							.locator('.dnd-tbody > div')
+							.first()
+							.locator('.dnd-td')
+							.allInnerTexts()
+					).toEqual(['["one","two","three"]', '']);
```

Comment:
I was expecting some more work to make default renderer work with this but after a deeper look, it's not needed